sqe-http-api/DTOs/ImagedObject.cs

```diff
@@ -24,4 +25,42 @@ public class ImagedObjectListDTO
     {
         public List<ImagedObjectDTO> imagedObjects { get; set; }
     };
+
+    public static class ImagedObjectIdFormat
+    {
+        /// <summary>
+        /// Creates a properly formatted imagedObjectId from the input parameters
+        /// </summary>
+        /// <param name="institution">Name of the institution that created the image</param>
+        /// <param name="catalogNumber1">Primary indexing number/identification</param>
+        /// <param name="catalogNumber2">Secondary indexing number/identification, if it exists</param>
+        /// <returns></returns>
+        public static string Serialize(string institution, string catalogNumber1, string catalogNumber2)
+        {
+            return institution + "-"
+                               + catalogNumber1
+                               + (string.IsNullOrEmpty(catalogNumber2)
+                                   ? ""
+                                   : "-" + catalogNumber2);
+        }
+
+        // Warning: I like the idea to create a readable imagedObjectID from the object's cataloguing information.
```

Comment:
We only created these mock IDs because the database didn't have them. Can't we just have the database handle them (even making them numeric)?

Reply:
I can just give you the image_catalog_id, that is the unique identifier for an institution-plate-fragment-side.

Reply:
So our ImagedObject id will be the image_catalog_id? This seems very clean and neat, yes. So we should change the ImagedObjectDTO's id field to a number?

Reply:
Yes.  Shall I make the change now, or would you rather we do it in a separate branch+pull request.

Reply:
Do it now.

Reply:
One little problem is that for each ImagedObjectDTO there are actually two image_catalog_id numbers (one for the recto and one for the verso).  I could just join those two with a hyphen "987-10267" and we would not have the decomposition problem with the other system.  What do you think?

Reply:
I think this brings us back into the parsing IDs. And although now IDs can't be misparsed (there are no hyphens in numbers), it's still a bit of a pain. Can you use the recto image ID?

Reply:
Yeah, I can do that.  

Perhaps we could make the object a bit more explicit so that it communicates sides better?  The endpoint for the imagedObject could default to recto+verso (and we give an optional for explicit request of recto or verso [and recto+verso?]).  The object you get back would have the image_catalog_id of the requested side as its id (default to recto), would have a property telling you the side (default to recto), and could also give you an oppositeSideId with the image_catalog_id of the opposite side (default to verso). Maybe this could be helpful as we build out better recto/verso support?  It would be the same DTO we are using now just with an added "side" property.

Reply:
I actually don't remember what we decided on recto/verso and images. Do we treat the verso as an entirely different image from the recto? If so, these are two separate ImagedObjects, each with its own side of course. And a link to the other one (otherSideId: ...)

Reply:
Sorry for the delay.  I am glad we have gone through this, and now I think I have come back to the original suggestion of using an id like IAA-994-1, which represents the object as it was designated by the museum.  The object IAA-994-1, for example, has recto and verso images (maybe the system somehow accommodates even more than that someday).  

So, let's leave the imagedObjectDTO as it is, I can add the necessary logic in the API/database to be able to search via this id (but avoid the composition/decomposition issues).  I think I originally liked what you did with that designation because it fits what the API user expects: a designation of an object, not of an image (or image series).

Reply:
In this case, I think we should just add this field to the database, shouldn't we?

Reply:
Yep, I will add it to the database.  Thanks for talking through this.

Reply:
Done